data/entity/npc/monster/undead/skeleton/skeletons.npcs.toml

```diff
@@ -147,3 +147,18 @@ id = 5365
 [stronghold_skeleton_shield]
 clone = "stronghold_skeleton_helmet"
 id = 5366
+
+[skeleton_catacombs]
+id = 7921
+hitpoints = 180
+att = 17
+str = 17
+def = 17
+attack_range = 5
```

Comment:
```suggestion
```
Are these ranged throwers or melee? If melee this isn't needed as default is `attack_range = 1`